relay-profiling/src/sample.rs

```diff
@@ -168,9 +168,6 @@ pub struct ProfileMetadata {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     transaction: Option<TransactionMetadata>,
 
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    measurements: Option<HashMap<String, Measurement>>,
```

Comment:
Does this moves where `measurements` are in the protocol? Is this a breaking change?

Reply:
It doesn't, it just changes when we parse them. This function is meant to parse metadata, so light parsing of most of the fields to check we should go further instead of dropping the profile.

I consider measurements to be data, so I'd want to parse them later, on a processing relay. It might not appear to make a ton of differences but it'll allow us to be able to show the SDK name/version in case there's malformed measurements data and will lower the load on pops, however slightly.

The protocol doesn't change, it stays where it is in the structure (we use the `flatten` option below to tell `serde` to not consider it part of a separate object).